packages/components/src/InfoBox.tsx

```diff
@@ -21,22 +20,11 @@ export const InfoBox = (props: InfoBoxProps): ReactNode => {
 
   const { colors } = useThemeFromContext();
 
-  return h(
-    PopupTrigger,
-    {
-      side,
-      content: div({ style: { padding: '0.5rem', width: 300 } }, [children]),
-    },
-    [
-      h(
-        Clickable,
-        {
-          'aria-label': 'More info',
-          tagName: 'span',
-          tooltip,
-        },
-        [icon(iconId, { size, style: { color: colors.accent(), cursor: 'pointer', ...style } })]
-      ),
-    ]
+  return (
+    <PopupTrigger content={<div style={{ padding: '0.5rem', width: 300 }}>{children}</div>} side={side}>
```

Comment:
We use [Prettier](https://prettier.io/) to automatically format code. This is how it formatted this line.

Prettier's ["Why Prettier?" page](https://prettier.io/docs/en/why-prettier) covers several reasons why this can be desirable.